framework/src/timeintegrators/CentralDifference.C

```diff
@@ -32,6 +32,7 @@ validParams<CentralDifference>()
 CentralDifference::CentralDifference(const InputParameters & parameters)
   : ActuallyExplicitEuler(parameters),
     _du_dotdot_du(_sys.duDotDotDu()),
+    _solution_older(_sys.solutionState(2)),
```

Comment:
If we use `enum`, `2` will become `older`. That help.

Reply:
See comment above. `solutionOlder()` still works, but I tried to keep things consistent here.